charts/v1.24/cray-hms-capmc/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: v1
+description: "Kubernetes resources for cray-hms-capmc"
+name: "cray-hms-capmc"
+home: "HMS/hms-capmc"
+version: 1.24.0
```

Comment:
![image](https://user-images.githubusercontent.com/77114568/144285640-302ee6fe-91b8-48ff-a7b2-73e7052ec81b.png)

We already have a 1.24.Z series of published CAPMC charts in artifactory. I think instead of revving the minor version, we should rev the patch version to `1.23.12`, so we don't conflict and start overriding artifacts in Algol60.

Reply:
@mjendrysik-hpe @nieuwsma

Reply:
Good catch, didn't think of that. Updated to 1.23.12.